src/InternPool.zig

```diff
@@ -546,6 +573,28 @@ pub const Key = union(enum) {
             s.flagsPtr(ip).alignment_wip = false;
         }
 
+        pub fn setInitsWip(s: @This(), ip: *InternPool) bool {
+            switch (s.layout) {
+                inline else => |layout| {
```

Comment:
Why use `inline else` here? The `Packed` case is the only one that needs special handling, we don't actually depend on it being comptime-known, and the rest of the body is trivial so it's fine to just duplicate it:
```zig
switch (s.layout) {
    .Packed => {
        const flag = &s.packedFlagsPtr(ip).field_inits_wip;
        if (flag.*) return true;
        flag.* = true;
        return false;
    },
    .Auto, .Extern => {
        const flag = &s.flagsPtr(ip).field_inits_wip;
        if (flag.*) return true;
        flag.* = true;
        return false;
    },
}
```